### ISSUE TITLE
Evaluate GitHub Apps

### ISSUE THREAD

**Issue by stevector** [pantheon-systems/terminus-build-tools-plugin#38]
I just got an email about GitHub Apps: https://developer.github.com/apps

> GitHub Apps are a new way to integrate with GitHub. With granular API permissions and built-in webhooks, you have more control over what you build.

@greg-1-anderson, this probably isn't a top priority right now but Apps seems like an appropriate mechanism for communicating between Pantheon and Github for tasks like commenting on PRs.

**Comment by stevector** [comment]
A key line from the marketing release:

> As first-class actors, GitHub Apps take actions themselves directly through the API—no user impersonation (or user seat) required

https://github.com/blog/2359-introducing-github-marketplace-and-more-tools-to-customize-your-workflow

I know the user account for commenting on PRs has been a pain point.

**Comment by greg-1-anderson** [comment]
Yes; GitHub allows service accounts by policy, but in fact, creating one usually quickly results in a ban / block, and you have to talk to support to identify and justify your service account.

I'll take a look at the apps feature and see if that can replace some of the things we're doing.
